pcap_ex.c

```diff
@@ -27,15 +27,15 @@ int
 pcap_ex_immediate(pcap_t *pcap)
 {
 #ifdef _WIN32
-    return pcap_setmintocopy(pcap, 1);
+	return pcap_setmintocopy(pcap, 1);
```

Comment:
I filed an issue to remind us of this: https://github.com/pynetwork/pypcap/issues/52